src/scripts/mapDiseaseOutbreakToAlerts.ts

```diff
@@ -0,0 +1,251 @@
+import { command, run } from "cmd-ts";
+import path from "path";
+import { getD2ApiFromArgs } from "./common";
+import {
+    RTSL_ZEBRA_ALERTS_DISEASE_TEA_ID,
+    RTSL_ZEBRA_ALERTS_EVENT_TYPE_TEA_ID,
+    RTSL_ZEBRA_ALERTS_NATIONAL_DISEASE_OUTBREAK_EVENT_ID_TEA_ID,
+    RTSL_ZEBRA_ALERTS_PROGRAM_ID,
+    RTSL_ZEBRA_ORG_UNIT_ID,
+    RTSL_ZEBRA_PROGRAM_ID,
+} from "../data/repositories/consts/DiseaseOutbreakConstants";
+import _ from "../domain/entities/generic/Collection";
+import { AlertD2Repository } from "../data/repositories/AlertD2Repository";
+import { NotificationD2Repository } from "../data/repositories/NotificationD2Repository";
+import { OptionsD2Repository } from "../data/repositories/OptionsD2Repository";
+import { Future } from "../domain/entities/generic/Future";
+import { getUserGroupsByCode } from "../data/repositories/utils/MetadataHelper";
+import { NotifyWatchStaffUseCase } from "../domain/usecases/NotifyWatchStaffUseCase";
+import {
+    getNotificationOptionsFromTrackedEntity,
+    getOutbreakKey,
+    mapTrackedEntityAttributesToAlertOutbreak,
+} from "../data/repositories/utils/AlertOutbreakMapper";
+import {
+    AlertData,
+    AlertSyncDataStoreRepository,
+} from "../data/repositories/AlertSyncDataStoreRepository";
+
+const RTSL_ZEBRA_DISEASE_TEA_ID = "jLvbkuvPdZ6";
```

Comment:
can we use codes instead?

Reply:
i'm only using the IDs because we can only filter by attribute ids and not codes. we could use the attribute codes here and then get the IDs using the api, but that seemed like an unneeded extra step

Reply:
ok, we are planning to fetch all metadata and cache on app load, so we will take care of this then.Please add a //TO DO : Fetch from metadata on app load